### ISSUE TITLE
Workflow should detect, check and use the installed alfred version.

### ISSUE THREAD

**Issue by ObiWahn** [deanishe/alfred-fuzzyfolders#12]
Hi,

this happens on Catalina (10.15.7) and with alfred 4. Let me know if I can help.

Update:
It looks a bit like bumping the version in the ff.py is enough to get it working.

**Comment by udiNur** [comment]
1. install and run [fixum](https://github.com/deanishe/alfred-fixum) workflow
1b. run it in Alfred by typing 'fixum' and select 'Fix Workflows'
2. right-click on the 'fuzzyfolders' workflow and choose 'open in finder'
3. edit the ff.py file on line 103 from "Alfred 3" to "Alfred 4", and save it [according to this issues](https://github.com/deanishe/alfred-fuzzyfolders/issues/11)

you good to go!

pls let my no if it works for you!

**Comment by ObiWahn** [comment]
Yes it works. That is what I meant by bumping the version in the `ff.py`. A good fix would make the workflow use the installed alfred version and emit some sort of error when the version requirement is not meat.
